indico/modules/events/registration/forms.py

```diff
@@ -250,3 +251,45 @@ class TicketsForm(IndicoForm):
                                           widget=SwitchWidget(),
                                           description=_('Allow users to download their ticket from the registration '
                                                         'summary page.'))
+
+
+class ParticipantsDisplayForm(IndicoForm):
+    """Form to customize the display of the participant list."""
+    json = JSONField()
+
+    def validate(self):
```

Comment:
why not use `def validate_json(self, field)`? That's the field-specific validation method